http-clients/feign-common/src/main/java/org/triplea/http/client/HttpClient.java

```diff
@@ -71,21 +71,16 @@ protected Response logAndRebufferResponse(
                     : super.logAndRebufferResponse(configKey, logLevel, response, elapsedTime);
               }
 
+              @FormatMethod
               @Override
               protected void log(
                   final String configKey, final String format, final Object... args) {
                 final String logMessage = String.format(format, args);
-                log.trace(configKey + ": " + logMessage);
+                log.info(configKey + ": " + logMessage);
               }
             })
         .logLevel(Logger.Level.BASIC)
-        .options(
-            new Request.Options(
-                DEFAULT_CONNECT_TIMEOUT_MS,
-                TimeUnit.MILLISECONDS,
-                DEFAULT_READ_TIME_OUT_MS,
-                TimeUnit.MILLISECONDS,
-                true))
+        .options(new Request.Options(DEFAULT_CONNECT_TIMEOUT_MS, DEFAULT_READ_TIME_OUT_MS))
```

Comment:
This `Request.Options` constructor is deprecated. 

```
Deprecated
please use Options(long, TimeUnit, long, TimeUnit, boolean)
```

Reply:
Fixed.